test/test_comments.sh

```
@@ -0,0 +1,189 @@
+GIT_EOF
+  chmod +x "$mock_dir/git"
+}
+
+setup_mocks_with_pr() {
+  local pr_reviews="$1"
+  local pr_issues="$2"
+
+  setup_mocks
+
+  printf '#!/usr/bin/env bash\ncase "$*" in\n' > "$mock_dir/gh"
+  printf '  *\"pulls?head=acme:feature-branch\"*)\n' >> "$mock_dir/gh"
```

Shellcheck (0.11.0)

[style] Consider using { cmd1; cmd2; } >> file instead of individual redirects.

(SC2129)